config.py

```diff
@@ -73,6 +73,7 @@ class Config(object):
     SHARE_SURVEY_ENABLED = bool(strtobool(os.getenv("SHARE_SURVEY_ENABLED", "True")))
     MAX_SHARED_SURVEY = int(os.getenv("MAX_SHARED_SURVEY", "50"))
     TECHNICAL_MESSAGE_ENABLED = bool(strtobool(os.getenv("TECHNICAL_MESSAGE_ENABLED", "True")))
+    UNDER_MAINTENANCE = os.getenv("UNDER_MAINTENANCE", False)
```

Comment:
Can change this to be more like the one above? make the default value a string and do the strtobool and bool stuff to get it into the right type.

It's a bit fiddly, but `os.getenv`  (https://docs.python.org/3/library/os.html#os.getenv) should have the return value and deafult value be a string so we should abide by this.